app/Console/Commands/SendDailyDigestNotifications.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Jobs\SendDailyDigestNotificationsJob;
+use App\Models\V2\Tasks\Task;
+use Illuminate\Console\Command;
+
+class SendDailyDigestNotifications extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'send-daily-digest-notifications';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Runs daily digest notifications for tasks';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        Task::isIncomplete()->get()->each(function (Task $task) {
```

Comment:
Tasks are pretty small, but there are 363 in `Task::isIncomplete()` in my current copy of prod. Better to do this in chunks instead of loading them all into memory at once. `chunkById` is what you want, and there are several examples of its use in the codebase, especially in the Commands code.